presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -129,6 +129,71 @@ static void addRuntimeMetricIfNotZero(
   }
 }
 
+// Add 'running time' metrics from Velox operator stats to Presto operator
+// stats.
+static void addRunningTimeOperatorMetrics(
+    protocol::OperatorStats& opOut,
+    protocol::TaskStats& prestoTaskStats,
+    const exec::OperatorStats& op) {
+  auto statName = fmt::format(
+      "{}.{}.runningAddInputWallNanos", op.operatorType, op.planNodeId);
```

Comment:
I feel it's okay for the statName to go without "running" prefix. Having "running" makes me wonder if it's part of addInput wall nanos

Reply:
I added prefix 'running' so these stats can be seen together in the list and easily searchable on query UI.
I think about it as runningAddInputWallNanos = <state><function><unit>?

Reply:
Makes sense to me